topi/python/topi/transform.py

```diff
@@ -268,3 +269,25 @@ def _compute(begin, *indices):
                         lambda *indices: _compute(begin_id, *indices), name="s%d" %i)
             for i, (out_shape, begin_id) in enumerate(zip(out_shapes, begin_ids))]
     # pylint: enable=cell-var-from-loop
+
+
+@tvm.target.generic_func
+def take(a_tuple, axis=0):
```

Comment:
always be consistent with numpy API https://docs.scipy.org/doc/numpy/reference/generated/numpy.take.html